gensim/test/test_word2vec.py

```diff
@@ -678,6 +678,16 @@ def test_sentences_should_not_be_a_generator(self):
     def testLoadOnClassError(self):
         """Test if exception is raised when loading word2vec model on instance"""
         self.assertRaises(AttributeError, load_on_instance)
+
+    def test_reset_from(self):
+        """Test if exception is raised when reset_from is used"""
```

Comment:
is there some reason that exception should be raised when reset_from is used? correct behavious is no exception. please add checks that the model is actually reset.